File: Vanilla/sources/zomboid/zombie/network/DBResult.java
package zombie.network;

import java.util.ArrayList;
import java.util.HashMap;

public class DBResult {
   private HashMap values = new HashMap();
   private ArrayList columns = new ArrayList();
   private String type;
   private String tableName;

   public HashMap getValues() {
      return this.values;
   }

   public String getType() {
      return this.type;
   }

   public void setType(String var1) {
      this.type = var1;
   }

   public ArrayList getColumns() {
      return this.columns;
   }

   public void setColumns(ArrayList var1) {
      this.columns = var1;
   }

   public String getTableName() {
      return this.tableName;
   }

   public void setTableName(String var1) {
      this.tableName = var1;
   }
}

File: Vanilla/sources/zdoc/media/lua/shared/Library/DBResult.lua
---@class DBResult : zombie.network.DBResult
---@field private values HashMap|Unknown|Unknown
---@field private columns ArrayList|Unknown
---@field private type String
---@field private tableName String
DBResult = {}

---@public
---@return HashMap|Unknown|Unknown
function DBResult:getValues() end

---@public
---@return String
function DBResult:getTableName() end

---@public
---@return String
function DBResult:getType() end

---@public
---@param arg0 ArrayList|Unknown
---@return void
function DBResult:setColumns(arg0) end

---@public
---@return ArrayList|Unknown
function DBResult:getColumns() end

---@public
---@param arg0 String
---@return void
function DBResult:setTableName(arg0) end

---@public
---@param arg0 String
---@return void
function DBResult:setType(arg0) end
